{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -81,7 +81,7 @@
 {% if cookiecutter.use_celery == 'y' %}
 ########## CELERY
 # In development, all tasks will be executed locally by blocking until the task returns
-CELERY_ALWAYS_EAGER = True
+CELERY_TASK_ALWAYS_EAGER = True
```

Comment:
I suggest setting [CELERY_EAGER_PROPAGATES](http://docs.celeryproject.org/en/latest/userguide/configuration.html#task-eager-propagates)=True here when using task_always_eager in local development so the exceptions will not be suppressed.

Reply:
Thanks for the advice, I think it makes sense.